broker-plugins/broker-query-engine/src/main/java/org/apache/qpid/server/query/engine/QueryEngine.java

```diff
@@ -112,6 +111,10 @@ public void setMaxBigDecimalValue(final BigDecimal maxBigDecimalValue)
         _maxBigDecimalValue = maxBigDecimalValue;
     }
 
+    public void setMaxQueryCacheSize(final int maxQueryCacheSize) {
```

Comment:
The curly bracket should be on new line.

Reply:
Fixed